javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,13 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  var newArr = [];
+  var filter = [];
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] !== null) {
+      newArr.push(values[i]);
+    } else {
+      filter.push(values[i]);
+    }
```

Comment:
You don't need to preserve the null values at all, you can just do nothing if its `null`